packages/database/src/migrations/20201201082830-AddProjectPalauOlangch-modifies-data.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+import { codeToId, insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const COUNTRY_CODE = 'PW';
+const PROJECT_CODE = 'olangch_palau';
+const PROJECT_NAME = 'MoH Project Olangch';
+const DASHBOARDGROUP_NAME = 'General';
+const DASHBOARDGROUP_CODE = 'PW_Olangch';
```

Comment:
Perhaps add a 'project'
![image](https://user-images.githubusercontent.com/32168886/100943600-aa41e380-3551-11eb-9423-a2ca59b2cd4e.png)

Reply:
Also I think most project level dashboards are called the name of the project (e.g. UNFPA), but that's not a big deal because Georgia told you otherwise

Reply:
I think standardisation is a good idea here, and i was my mistake. Will update.